apps/spruce/src/gql/generated/types.ts

```diff
@@ -6203,6 +6200,50 @@ export type AdminSettingsQuery = {
     __typename?: "AdminSettings";
     banner?: string | null;
```

Comment:
i think i wanna wrap these into an announcements struct at some point too that would make sense right?

Reply:
You can but it's not necessary, the point of `transformers` is to transform data into different formats